crates/gas/src/lib.rs

```diff
@@ -54,8 +54,14 @@ const COMPILE_GAS_PER_BYTE: u64 = 1_955;
 const PARALLEL_GAS_DIVIDER: u64 = 10;
 const WASM_CODE_VALIDATION_GAS_PER_BYTE: u64 = 67;
 const WRAPPER_TX_VALIDATION_GAS: u64 = 3_245_500;
+// There's no benchmark to calculate the cost of storage occupation, so we
+// define it as the cost of storage latency (which is needed for any storage
+// operation and it's based on actual execution time), plus the same cost
+// multiplied by an arbitrary factor that represents the higher cost of storage
+// space as a resource. This way, the storage occupation cost is not completely
+// free-floating but it's tied to the other costs
 const STORAGE_OCCUPATION_GAS_PER_BYTE: u64 =
-    100_000 + PHYSICAL_STORAGE_LATENCY_PER_BYTE;
+    PHYSICAL_STORAGE_LATENCY_PER_BYTE * (1 + 10);
```

Comment:
why did you write the argument as `(1 + 10`? Do each of these numbers account for some different effect?

Reply:
They do: the 1 represent the actual cost of storage latency, the 10 component instead is just an arbitrary multiplier that I use to compute an actual storage occupation gas cost based on the storage latency. They are not really tied but we cannot leave the storage occupation gas cost free-floating because we'd end up with the same issue this Pr tries to solve. So we need a way to tie that cost to some other gas cost that is actually based on a benchmark and the storage latency is the closest one

Reply:
cool would you mind describing this a bit in a comment / docstring?

Reply:
It's described right above

Reply:
Ok I think I was having trouble understanding the comment at first and was seeking clarity, but I think it's fine now. Thanks.